### PR TITLE
fixes issue #458

### PR DESCRIPTION
**Pull request by butterflyskip**
Did not add Launchpad or GNU Savannah.  I can add it later, if you like.

### CODE REVIEW

File: _episodes/13-hosting.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ option below.
 
 The third option is to use a public hosting service like
 [GitHub](https://github.com), [GitLab](https://gitlab.com),
-[BitBucket](https://bitbucket.org), or [SourceForge](http://sourceforge.net).
+[BitBucket](https://bitbucket.org).
```
Comment (iglpdc) [inline _episodes/13-hosting.md]: Could you add an "or" before Bitbucket?

**Comment by iglpdc** [comment]
Thanks, @butterflyskip ! Could you please address my comment above?

File: _episodes/13-hosting.md (hunk 1 above)
Comment (butterflyskip) [reply _episodes/13-hosting.md]: Done.

**Comment by iglpdc** [comment]
Thanks, @butterflyskip !

**Pull request merged**